src/caret.js

```diff
@@ -1,43 +1,46 @@
-let getCaret;
-let setCaret;
+export default function installCaret(_document = document) {
```

Comment:
Good catch. I had a file waiting to be committed, the tests no less. So yes the tests do use this option.